v2/apiserver/internal/authz/mongodb/role_assignments_store.go

```diff
@@ -40,7 +40,7 @@ func (r *roleAssignmentsStore) Grant(
 		&options.FindOneAndReplaceOptions{
 			Upsert: &tru,
 		},
-	); res.Err() != nil {
+	); res.Err() != nil && res.Err() != mongo.ErrNoDocuments {
```

Comment:
TIL that an "upsert" returns a `mongo.ErrNoDocuments` error on a successful _insert_ (but not on an _update_). I want to suppress that because it's not actually any kind of a problem.